.github/workflows/publish-draft-releases.yml

```diff
@@ -305,15 +306,16 @@ jobs:
         if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
         name: mv and chmod manta-base
         run: |
-          mv ${{ github.workspace }}/manta-base $HOME/.local/bin/
+          ls -ahl ${{ github.workspace }}/
+          mv ${{ github.workspace }}/manta $HOME/.local/bin/manta-base
```

Comment:
@stechu after the artifact is unzipped the name is `manta` . So we rename it to `manta-base` when we move it and we can continue without other changes.